cmd/aws/lambda/autoscale/main.go

```diff
@@ -63,7 +63,11 @@ func init() {
 			panic(err)
 		}
 
-		kinesisDownscaleDatapoints = int32(dps)
+		if dps <= 360 && dps >= 1 {
```

Comment:
The max value can be any integer, so I think borrowing this logic from your first commit may work better:
```go
if dps <= math.MaxInt32 && dps >= 1 {
```

Same for the other line, too.

Reply:
sounds good, adjusted - i was basing off of the readme-defined maxes

Reply:
I think that README might have been accurate at one time, but not anymore. It could use an update.